lms/djangoapps/grades/api/v1/views.py

```diff
@@ -182,7 +182,10 @@ def _iter_user_grades(self, course_key):
         Returns:
             An iterator of CourseGrade objects for users enrolled in the given course.
         """
-        enrollments_in_course = enrollment_data.get_user_enrollments(course_key)
+        enrollments_in_course = CourseEnrollment.objects.filter(
```

Comment:
Is `enrollment_data` an unused import now?

Reply:
There's still one more usage of it.  I have a related PR in which that import will go away.